packages/kubernetes-1.18/kubelet-exec-start-conf

```diff
@@ -2,14 +2,12 @@
 ExecStart=
 ExecStart=/usr/bin/kubelet \
 {{~#unless settings.kubernetes.standalone-mode}}
-    --cloud-provider aws \
     --kubeconfig /etc/kubernetes/kubelet/kubeconfig \
 {{~#if (eq settings.kubernetes.authentication-mode "tls")}}
     --bootstrap-kubeconfig /etc/kubernetes/kubelet/bootstrap-kubeconfig \
 {{~/if}}
-{{~else}}
-    --cloud-provider "" \
 {{~/unless}}
+    --cloud-provider {{settings.kubernetes.cloud-provider}} \
```

Comment:
We need the old logic here so that we pass an empty string to cloud-provider if standalone mode is enabled.